website/static/js/profile.js

```diff
@@ -346,7 +364,7 @@ var NameViewModel = function(urls, modes, preventUnsaved, fetchCallback) {
     self.middle = koHelpers.sanitizedObservable().extend({trimmed: true});
     self.family = koHelpers.sanitizedObservable().extend({trimmed: true});
     self.suffix = koHelpers.sanitizedObservable().extend({trimmed: true});
-
+    
```

Comment:
Sounds like something significant, I shall settle this situation by sequestering those spaces, tout suite!